docs/themes/theme_settings.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Writing Theme Settings
+---
+
+You can provide users with theme settings, configurable parts of your theme that help users customize the theme to their projects. Examples of theme settings include configurable element colors, image asset uploads, footer content.
+
+You create the form fields for your theme settings in `config/theme.ini`. Start with `[config]` as the heading under the `[info]` metadata. The `[config]` heading begins a list of theme settings; you do not need to include it over each individual setting.
+
+```
+[info]
+name = "Theme Name"
+,,,
+
+[config]
+```
+
+Each theme setting is a form element, so you begin by labeling your setting, providing it with a unique lowercase name, and adding it to the list of elements.
+
+```
+[config]
+elements.my_theme_setting.name = "my_theme_setting"
+elements.my_theme_setting.options.label = "My Theme Setting"
+```
+
+Next you need to define your theme setting's form element type.
+
+The available form element types from Zend framework are found in `vendor\zendframework\zend-form\src\Element`. In the config file, reference the element type with the path `Zend\Form\Element\[Type]`. The following example defines the theme setting as a checkbox.
```

Comment:
probably want to point people at ZF documentation rather than the files in the install: https://docs.zendframework.com/zend-form/element/intro/

plus I'll nitpick here on capitalization of "Framework"